src/components/FooterGrid/index.module.scss

```diff
@@ -5,11 +5,17 @@
   grid-column-gap: 28px;
   grid-row-gap: 28px;
 
+  padding-right: 20px;
+  padding-left: 20px;
+
   @media (min-width: 768px) {
     justify-self: center;
     grid-row-gap: 56px;
     grid-template-columns: 1fr 1fr;
     max-width: 80%;
+
+    padding-right: 0;
+    padding-left: 0;
```

Comment:
I'm not sure if this is actually necessary. But if it is, you can have a look at `base.scss` and see what Gabx did for `--post-max-width` and `--post-body-max-width`